custom_components/ramses_cc/services.yaml

```diff
@@ -133,33 +133,27 @@ set_system_mode:
             - custom
 
     period:
-      name: Days
+      name: Period (days)
       description: >-
-        Optional. A period of time in days; used only with away, day_off, or custom.
+        Optional. A period of time in days; used only with away, day_off, day_off_eco or custom.
         The system will revert to auto at midnight (up to 99 days, today is day 1).
-      default: 1
+      default:
+        days: 28
       example: '{"days": 28}'
       selector:
-        number:
```

Comment:
As the service schema uses the standard time period schema for this field, any number entered will be interpreted in seconds not days. Rather than change the schema and potentially break existing calls which take advantage of this it seemed safer to instead switch to the duration UI selector (even if the hours/minutes/seconds are ignored). 

An alternative to this would be the `object` selector which may be preferable as that may make it clearer that only days are accepted?

Reply:
> I am very willing to make breaking changes, if that is for the greater good.

It would probably make most sense to have the same `duration` and `until` fields here as for `set_zone_mode`. Perhaps something for the refactor branch? Keeping backwards compat for `period` probably wouldn't be too hard either but it sounds like it's perhaps not worth the effort.

> I could never get that UI selector to work - have you tested it? I think we shoudl use it if we can.

Seemed fine to me. If there are fewer limitations in the underlying system it probably makes perfect sense to expose full flexibility anyway.

Reply:
Yes - keep breaking changes minimal for **master** - I am working towards making it the new **stable** branch (dodgy workflow).

But be happy to do the 'right' thing for **refactor** - even if breaking change (keep a track of them).